### PR TITLE
 Don't make Solr commits during bulk operations

### LINKED ISSUE #598: Never commit from code, only from Solr, when doing bulk operations
Reducing the commit spam on the solr server. Might help especially when doing a full re-index.

### PR DESCRIPTION
**Pull request by eliotjordan**
Closes #598 

~~Don't merge until https://github.com/pulibrary/pul_solr/pull/131 is merged, deployed, and tested.~~ Complete.

### CODE REVIEW

**Comment by eliotjordan** [comment]
Waiting to merge this until some changes are made in Figgy. Solr commits should probably only be suppressed during bulk operations. Map lab staff rely on quick feedback to evaluate updates in Figgy. Enabling this PR as is will require a long wait for updates to reflect in Pulmap.

File: app/services/geoblacklight_event_processor/processor.rb [hunk 1]
```diff
@@ -19,6 +19,10 @@ def doc
       event['doc']
     end
 
+    def bulk?
```
Comment (jrgriffiniii) [inline app/services/geoblacklight_event_processor/processor.rb]: For the purposes of my own understanding, this depends upon a Figgy server environment var. specifying that only bulk updates are used, correct?

Please see https://github.com/pulibrary/figgy/blob/master/app/services/event_generator/geoblacklight_event_generator.rb#L65 for reference.

Comment (eliotjordan) [reply app/services/geoblacklight_event_processor/processor.rb]: Yes. To reindex Pulmap, you would start a console session with `BULK=true` and run the the reindexer service.

https://github.com/pulibrary/figgy/blob/master/app/services/geo_resource_reindexer.rb

There's not a Pulmap reindex job right now, but there probably should be.

Comment (jrgriffiniii) [reply app/services/geoblacklight_event_processor/processor.rb]: If this is indeed the case, then I can create an issue in https://github.com/pulibrary/princeton_ansible before merging this (unless this should be controlled in the Capistrano config. for Figgy).

Comment (eliotjordan) [reply app/services/geoblacklight_event_processor/processor.rb]: Oh, sorry I misread your comment. You only set the BULK var when doing a full reindex.

Comment (eliotjordan) [reply app/services/geoblacklight_event_processor/processor.rb]: Normal atomic updates and deletes are unchanged.

**Pull request merged**